modules/oe_content_publication/config/install/core.entity_form_display.node.oe_publication.default.yml

```diff
@@ -72,41 +98,101 @@ content:
       maxlength:
         maxlength_js: 170
         maxlength_js_label: 'Content limited to @limit characters, remaining: <strong>@remaining</strong>'
+  oe_departments:
+    weight: 10
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: skos_concept_entity_reference_autocomplete
+    region: content
   oe_documents:
-    weight: 7
+    weight: 8
     settings:
       match_operator: CONTAINS
       size: 60
       placeholder: ''
+      match_limit: 10
     third_party_settings: {  }
     type: entity_reference_autocomplete
     region: content
+  oe_publication_contact:
+    weight: 14
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: entity_reference_revisions_autocomplete
+    region: content
+  oe_publication_country:
+    weight: 12
+    settings:
+      match_operator: CONTAINS
+      match_limit: 10
+      size: 60
+      placeholder: ''
+    third_party_settings: {  }
+    type: skos_concept_entity_reference_autocomplete
+    region: content
   oe_publication_date:
-    weight: 3
+    weight: 16
     settings: {  }
     third_party_settings: {  }
     type: datetime_default
     region: content
+  oe_publication_last_updated:
+    weight: 17
+    settings: {  }
+    third_party_settings: {  }
+    type: datetime_default
```

Comment:
In all content types select lists are used to set date (`type: datetime_datelist` or `type: daterange_datelist`) except News. @ademarco what widget should be used for date field or in different CT different widgets will be used?

Reply:
I changed the date related selector